docs/cep/query-guide/functions/json/getLong.md

```diff
@@ -38,8 +38,28 @@ Given a JSON object like `{'name' : 'John', 'age' : 23}`, the function `json:get
 ## Example 3
 
 ```sql
-@info(name = 'query1')
+@info(name = 'query3')
 json:getLong(json,'$.name')
 ```
 
 For a JSON object like `{'name' : 'John', 'age' : 23}`, the function `json:getLong(json,'$.name')` searches for a long value associated with the key `name`. In this case, it returns `null` as there is no matching long at `$.name`.
+
+## Example 4
+
+```sql
+CREATE STREAM PersonStream (json string);
```

Comment:
@koshy-thomas and @Luba89, this one won't validate, cannot figure out why.

Reply:
@diana-macrometa its bool not boolean, just change boolean to bool  in the below line of create sink MarriedPersonStream to: 
CREATE SINK STREAM MarriedPersonStream (name string, married bool);